docs/manual/overview.rst

```diff
@@ -28,7 +28,8 @@ In Spox, after calling an operator constructor type inference is immediately run
 Spox provides constructor functions for all operators defined by the ONNX specification.
 This includes both the `ai.onnx <https://github.com/onnx/onnx/blob/main/docs/Operators.md>`_ and `ai.onnx.ml <https://github.com/onnx/onnx/blob/main/docs/Operators-ml.md>`_ domain.
 
-Constructors are autogenerated with the ``src/generate.py`` script based on the list of operator schemas provided by ``onnx``. The modules with them may be imported from ``spox.opset.ai.onnx.*``, at a given version of the opset.
+Constructors are autogenerated with the ``codegen/generate.py`` script based on the list of operator schemas provided by ``onnx``.
```

Comment:
I would opt for a less specific and more commonly seen name, like `tools/`, maybe `tools/codegen/`. Then we'd have `tools/generate.py` & `tools/templates/` (or `tools/codegen_templates/`). `codegen` might evoke strong feelings :)